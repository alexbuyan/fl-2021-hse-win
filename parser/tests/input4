4 \
1 - 0 \
2 - 1 \
3 - 3 \
4 - 2 \
1 -> 1 : "дед мороз", "\"", "\\" \
1 -> 2 : "северный" \
1 -> 4 : "олень" \
2 -> 2 : "дед мороз", "\"", "\\"\
2 -> 3 : "олень" \
2 -> 4 : "северный" \
3 -> 3 : "дед мороз", "\"", "\\" \
3 -> 4 : "северный", "олень" \