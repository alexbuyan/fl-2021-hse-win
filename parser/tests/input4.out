Alphabet:
	{'олень', '\\"', 'дед мороз', 'северный', '\\\\'}
States:
	(name: 1, type: start)
	(name: 2, type: normal)
	(name: 3, type: terminal)
	(name: 4, type: stock)
Edges:
	(from: 1, to: 1, moves: дед мороз,\",\\)
	(from: 1, to: 2, moves: северный)
	(from: 1, to: 4, moves: олень)
	(from: 2, to: 2, moves: дед мороз,\",\\)
	(from: 2, to: 3, moves: олень)
	(from: 2, to: 4, moves: северный)
	(from: 3, to: 3, moves: дед мороз,\",\\)
	(from: 3, to: 4, moves: северный,олень)
